cli/image-scanning-by-using-the-roxctl-cli.adoc

```diff
@@ -0,0 +1,68 @@
+[id="image-scanning-by-using-the-roxctl-cli"]
+:_content-type: ASSEMBLY
+= Image scanning by using the roxctl CLI
+include::modules/common-attributes.adoc[]
+:context: image-scanning-by-using-the-roxctl-cli
+
+toc::[]
+
+[role="_abstract"]
+You can use the `roxctl` CLI to scan images stored in image registries, including cluster local registries such as the {ocp} integrated image registry.
+
+[id="scanning-images-in-a-remote-cluster_{context}"]
+== Scanning images using a remote cluster
+
+To scan images from cluster local registries using a remote cluster, specify the appropriate cluster using the delegated scanning configuration or use the cluster parameter described in the following procedure.
+
+[IMPORTANT]
+====
+For more information on how to configure delegated image scanning, see xref:../operating/examine-images-for-vulnerabilities.adoc#configuring-delegated-image-scanning_examine-images-for-vulnerabilities[Configuring delegated image scanning].
+====
+
+.Procedure
+
+* Run the following command to scan the specified image in a remote cluster:
++
+[source,terminal,subs="+quotes"]
+----
+$ roxctl image scan --image=_<image_registry>_/_<image_name>_ --cluster=_<cluster_name>_ [flags]<1>
+----
++
+--
+<1> For `<image_registry>`, specify the registry where the image is located, for example, `image-registry.openshift-image-registry.svc:5000/`.
++
+For `<image_name>`, specify the name of the image you want to scan, for example, `default/image-stream:latest`.
++
+For `<cluster_name>`, specify the name of the remote cluster, for example, `remote`.
```

Comment:
I didn't catch this before, but this parameter can be the cluster `name` or cluster `id`